### DISCUSSION TITLE
PKG version

### DISCUSSION THREAD

**Discussion by tdtooke** [fastfetch-cli/fastfetch#447] [General]
I recently decided to start following this app a lot more closely.  So I made my own fastfetch-git pkgbuild.  One odd thing.  I know the current release is 1.11.0 but my pkgver function returned: "1.10.3.r58.g7d0a1c8".  I don't want to put any sort of priority on this question at all so I didn't put in in Q&A but that really struck me as odd, the r part afterwards is pretty standard for when you get the version from a function but 1.10.3 is less than 1.11.0 so....

Edit: Funny, that pkg already exists.  I always thought that would be considered a duplicate.  If I had known that I would have put Scidvspc-svn up there myself a long time ago...

**Comment by LinusDierheimer** [comment]
This happens because the tagging is done only on the main branch by GH actions. I merged the tags in the dev branch, so this does work now.
